resources/providers/awslib/account_provider.go

```diff
@@ -65,3 +82,63 @@ func listAccounts(ctx context.Context, client organizations.ListAccountsAPIClien
 	}
 	return accounts, nil
 }
+
+type organizationalUnitInfo struct {
+	id   string
+	name string
+}
+
+func getOUInfoForAccount(ctx context.Context, client organizationsAPI, cache map[string]string, accountId *string) (organizationalUnitInfo, error) {
+	paginator := organizations.NewListParentsPaginator(client, &organizations.ListParentsInput{ChildId: accountId})
+	for paginator.HasMorePages() {
+		o, err := paginator.NextPage(ctx)
+		if err != nil {
+			return organizationalUnitInfo{}, err
+		}
+		if len(o.Parents) == 0 {
+			continue
+		}
+
+		parent := o.Parents[0]
```

Comment:
might worth adding some context on why we're picking the first one

Reply:
Added a few comments